tests/e2e/script_test.go

```diff
@@ -95,7 +107,25 @@ func TestE2EScript(t *testing.T) {
 					switch testCommand {
 					case "APPLY":
 						applyObject(h, obj)
-						create.WaitForReady(h, obj)
+						_, noWait := obj.Object["NO_WAIT"]
```

Comment:
Any reason not to have a different command APPLY-NO-WAIT (to mirror DELETE-NO-WAIT).

My gut was that we might end with commands like `APPLY(wait=false)` or something like that, but I wanted to see how it evolved.

In that spirit though, let's see where it goes... it's a test and doesn't escape our code!

Reply:
Ah yes I can change to `APPLY-NO-WAIT` 👍🏼